test/CloudBoilerplateNet.Tests/Fixtures/CachedDeliveryClient/item.json

```diff
@@ -0,0 +1,221 @@
+﻿{
```

Comment:
Is it possible to use more semantic file name?

My tip:
item_with_null_identifiers.json

Reply:
Sure, good idea.